### PR TITLE
Illustration guide

### PR DESCRIPTION
**Pull request by chrislwk**
# Checklist

- I have spellchecked my changes.
- I have updated the API Documentation CHANGELOG where applicable.
- I have added my guides to Welcome.md where applicable
- I have added my guides to toc.json where applicable.

### CODE REVIEW

File: docs/illustrations/Basics.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+# Basics
+
+## What is an illustration?
+
+An illustration is a document which models a product over time. They are predominantly used for pension products, which must be produced according to regulation. 
+
+An accumulation illustration models the growth of a product based on contributions or transfers added to the product, expected investment growth and the impact of fees. Growth rates are governed by legislation, whilst fees should represent those which are applicable to the customer. Figures are an estimation based on the details provided and the value of the product could be more or less than what has been projected.
+
+Drawdown illustrations are similar to accumulation illustrations, but these cover at-retirement options where funds can be taken from a pension. Investment growth and fees work in the same way compared to an accumulation illustration, however income (instead of contributions) may be modelled in the illustration.
```
Comment (LukeBillo) [inline docs/illustrations/Basics.md:13]: If we're going to give information about drawdown illustrations here, I think it's worth mentioning that we don't support them at the end of this paragraph (as well as in the FAQs, as you've already done)

File: docs/illustrations/Basics.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## How to produce an illustration
+
+The below diagram represents the process flow for generating an illustration.
+
+```mermaid
+sequenceDiagram
+    Tenant->>WealthKernel: Request illustration (/sipp-illustrations)
+    WealthKernel->>Tenant: resource ID returned (illustrationId)
+    WealthKernel-->>WealthKernel: Illustration calculation
```
Comment (LukeBillo) [inline docs/illustrations/Basics.md:25]:
```suggestion
    WealthKernel-->>WealthKernel: Illustration generation
```

File: docs/illustrations/FAQs.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+At present, the illustration service only supports accumulation illustrations. Illustrations for drawdown and drawdown transfer will be added in the future. 
+
+## Does the illustration service support fee tiering?
+
+Yes, fee tiers can be accommodate, as per the example in the API documentation.
+
+## Is it possible to add adviser, DFM and product provider charges in one illustration?
+
+Yes, this combination of fees can be added, as per the example in the API documentation.
```
Comment (LukeBillo) [inline docs/illustrations/FAQs.md:17]: We should probably put some sort of blurb here about where each type of fee should go.
If I remember correctly, adviser fees must go under `adviser`, whilst DFM and platform fees must go under `provider`?

File: docs/illustrations/Basics.md (hunk 1 above)
Comment (chrislwk) [reply docs/illustrations/Basics.md:13]: Added a new section to show the supported scope and pulled the accumulation inputs section up into this

File: docs/illustrations/FAQs.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## Does the illustration service support fee tiering?
+
+Yes, fee tiers can be accommodated, as per the example in the API documentation.
+
+## Is it possible to add adviser, DFM and product provider charges in one illustration?
+
+Yes, this combination of fees can be added, as per the example in the API documentation. 
+
+Adviser fees can be added into the adviser section ofthe API request. DFM and product provider fees can be added into the 'provider' section of the API request.
```
Comment (LukeBillo) [inline docs/illustrations/FAQs.md:19]:
```suggestion
Adviser fees can be added into the adviser section of the API request. DFM and product provider fees can be added into the 'provider' section of the API request.
```

**Review by LukeBillo — APPROVED** [review]

**Pull request merged**